### PR TITLE
Liberland

### PR DESCRIPTION
**Pull request by N1ghtStorm**
Adds Liberland logic to relayer

### CODE REVIEW

File: relayer/src/substrate/types.rs [hunk 1]
```diff
@@ -33,6 +33,7 @@
 use bridge_types::H256;
 use codec::IoReader;
 use common::{AssetId32, PredefinedAssetId};
+pub use liberland_gen::{liberland_runtime, LiberlandExtrinsicParams};
```
[the remark this answers is not in the document]
Comment (N1ghtStorm) [reply relayer/src/substrate/types.rs]: fixed

[the remark this answers is not in the document]
Comment (N1ghtStorm) [reply relayer/src/substrate/types.rs]: fixed

**Review by vovac12 — APPROVED** [review]

**Review by Alexey-N-Chernyshov — APPROVED** [review]
It seems that it is unable to build with rust `stable`. Should we change `rust-toolchain`?
And I'm worried that we don't have tests.

**Comment by N1ghtStorm** [comment]
> It seems that it is unable to build with rust `stable`. Should we change `rust-toolchain`? And I'm worried that we don't have tests.

I think we should make e2e tests that could run on dev environment that will check the relayer as well

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+# Overview
+
+Relayer for Sora2 bridges
+
+### Build
+
+```sh
+cargo b -r
+```
+# Run Relayers
```
Comment (vovac12) [inline README.md:12]: You can also add some info about how to generate `signer` seed and public key for relayer registration

**Pull request merged**